doc/nrf/ug_nrf5340.rst

```diff
@@ -81,20 +77,21 @@ The OpenAMP library uses the IPM SHIM layer, which in turn uses the IPC driver i
 Available samples
 *****************
 
-nRF5340 samples consist of two separate images: one that runs on the network core and one that runs on the application core.
+nRF5340 samples consist of two separate images: one that runs on the network core and one that runs on the application core. Most of the samples which support nRF53 are not dedicated exclusively for this device but can also be build for other chips that have the radio peripheral e.g. nRF52 family. The following subsections briefly list these samples and focus more on those, which run exclusively on nRF53.
```

Comment:
```suggestion
nRF5340 samples usually consist of two separate images: one that runs on the network core and one that runs on the application core. Most of the samples which support nRF53 are not dedicated exclusively for this device but can also be build for other chips that have the radio peripheral e.g. nRF52 family. The following subsections briefly list these samples and focus more on those, which run exclusively on nRF53.
```
I think we support BLE NS images and this case application part consists of two images (spm + app).

Reply:
Good point, fixed.